ui/components/ui/box/box.stories.js

```diff
@@ -583,7 +583,7 @@ export const ResponsiveProps = () => {
         padding={[2, 4]}
         gap={[2, 4]}
         display={['flex', null, null, 'none']}
-        flexDirection={['column', 'row']}
+        flexDirection={['column', 'column', 'row']}
```

Comment:
Just realized we should probably be using the design-system consts here. Old mistake from me so my bad. But not blocking

e.g. `display={[DISPLAY.FLEX, null, null, DISPLAY.NONE]}`